llmfoundry/callbacks/model_gauntlet_callback.py

```diff
@@ -20,6 +21,13 @@ class Weighting(Enum):
     LOG_SAMPLE_SZ = 3
 
 
+def get_in_memory_logger(logger: Logger):
+    for lg in logger.destinations:
+        if isinstance(lg, InMemoryLogger):
+            return lg
+    raise Exception("Couldn't find InMemoryLogger in logger destinations!")
```

Comment:
`Exception` should almost never be used. In thise case, you should throw a `ValueError`.